models/core/core_models.yml

```diff
@@ -283,6 +283,81 @@ models:
     - name: tuva_last_run
       description: '{{ doc("tuva_last_run") }}'
 
+  - name: core__immunization
+    description: >
+      The immunization table contains information on immunizations administered to patients,
+      including the vaccine code, description, and administration date.
+    config:
+      schema: |
+        {%- if var('tuva_schema_prefix',None) != None -%}{{var('tuva_schema_prefix')}}_core
+        {% else %}core{%- endif -%}
+      alias: immunization
+      tags: core
+      materialized: table
+    columns:
+    - name: immunization_id
+      description: '{{ doc("immunization_id") }}'
+      tests:
+        - unique:
+            config:
+              severity: warn
+    - name: person_id
+      description: '{{ doc("person_id") }}'
+      tests:
+        - not_null
+    - name: patient_id
+      description: '{{ doc("patient_id") }}'
+    - name: encounter_id
+      description: '{{ doc("encounter_id") }}'
+    - name: source_code_type
+      description: '{{ doc("source_code_type") }}'
+      meta:
+        terminology: https://github.com/tuva-health/the_tuva_project/blob/main/seeds/terminology/terminology__code_type.csv
```

Comment:
Same note for terminology links here. See example in:
https://github.com/tuva-health/tuva/pull/1014